doc/tools/apigen.py

```diff
@@ -207,7 +207,11 @@ def _parse_module_with_import(self, uri):
         classes : list of str
             A list of (public) class names in the module.
         """
-        mod = __import__(uri, fromlist=[uri])
+        try:
+            mod = __import__(uri, fromlist=[uri])
+        except ImportError:
+            # fix ImportError for submodules in Python 3
+            mod = __import__(uri, fromlist=[])
```

Comment:
strange.  I still get the same error without the space.  However, If I replace the relative imports like `..` with absolute imports from `nibabel.` then it works fine for me.  That doesn't really make sense, though, because the same sorts of relative imports are working fine from other files such as the ones in /benchmarks.